Development/pcm_alt.yml

```diff
@@ -0,0 +1,45 @@
+name: ConsIndShock
+
+states:
+  - name: m
+    long_name: market resources
+    domain: [0, .inf]
+
+controls:
+  - name: c
+    long_name: consumption
+    domain: [0, .inf]
+  - name: stigma
+    long_name: risky portfolio share
+    domain: [0, 1]
+
+distributions:
+  perm: !MeanOneLogNormal
+    std: std_perm
+  tran: !MeanOneLogNormal
+    std: std_tran
+  risky: !LogNormal
+    mean: mean_risky
+    std: std_risky
+
+parameters:

```

Comment:
`calibration` and `parameters` convey the same information, so we can get rid of `calibration`